src/pages/navod.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Link from 'next/link';
+import { NextPage } from 'next';
+import { getRoutes } from '../lib/routes';
+
+const { previousRoute } = getRoutes('/navod');
+
+const Stiahnut: NextPage<{}> = () => {
```

Comment:
Toto by mal byt asi `Navod` Ale inac nedame uz cele toto prec a nelikneme to co uz je na navodoch? Niekde na staging to tusim maju

Reply:
Suhlasim, dam tam link na https://staging.navody.digital/zivotne-situacie/elektronicke-podanie-danoveho-priznania